client/src/pages/reports/Compact.js

```diff
@@ -0,0 +1,1012 @@
+                className="reportField"
+              />
+              <CompactRow
+                label="advisors"
+                content={getAdvisorAttendees()}
+                className="reportField"
+              />
+              <CompactRow
+                label={Settings.fields.task.subLevel.longLabel}
+                content={getTasksAndAssessments()}
+                className="reportField"
+              />
+              {report.cancelled ? (
+                <CompactRow
+                  label="cancelled reason"
+                  content={utils.sentenceCase(report.cancelledReason)}
+                  className="reportField"
+                />
+              ) : null}
+              {isActiveField("Workflow") && report.showWorkflow() ? (
+                <CompactRowReportWorkflow
+                  workflow={report.workflow}
+                  compactStyle={WORKFLOW_STYLE}
+                  className="reportField"
+                />
+              ) : null}
+              {report.reportText ? (
+                <CompactRow
+                  label={Settings.fields.report.reportText}
+                  content={parseHtmlWithLinkTo(report.reportText)}
+                  className="reportField"
+                />
+              ) : null}
+              {Settings.fields.report.customFields ? (
+                <ReadonlyCustomFields
+                  fieldsConfig={Settings.fields.report.customFields}
+                  values={report}
+                  vertical
+                  compactStyle=";" // avoid falsy empty string for compact condition checks in deeper components
+                />
+              ) : null}
+            </CompactTable>
+            <CompactReportFooterContent report={report} />
+          </CompactView>
+        </>
+      )}
+    </Formik>
+  )
+
+  function returnToDefaultPage() {
+    history.push(`/reports/${report.uuid}`)
+  }
+
+  function printReport() {
+    if (typeof window.print === "function") {
+      window.print()
+    } else {
+      alert("Press CTRL+P to print this report")
+    }
+  }
+
+  function getReportTitle() {
+    return (
+      <>
+        Engagement of{" "}
+        <LinkTo
+          modelType="Person"
+          model={Report.getPrimaryAttendee(
+            report.attendees,
+            Person.ROLE.PRINCIPAL
+          )}
+        />{" "}
+        by{" "}
+        <LinkTo
+          modelType="Person"
+          model={Report.getPrimaryAttendee(
+            report.attendees,
+            Person.ROLE.ADVISOR
+          )}
+        />
+        <br />
+        on{" "}
+        {moment(report.engagementDate).format(
+          Report.getEngagementDateFormat()
+        )}{" "}
+        at{" "}
+        {report.location && (
+          <LinkTo modelType="Location" model={report.location} />
+        )}
+      </>
+    )
+  }
+
+  function getReportSubTitle() {
+    const timeToShow = report.isDraft()
+      ? moment(report.updatedAt)
+      : moment(report.releasedAt)
```

Comment:
When report is submitted but not published (i.e. pending approval) date is displayed as `Authored at Invalid date` because `report.releasedAt` is `null`. I suggest displaying `report.updatedAt` when report is not published.

```
    const timeToShow = !report.isPublished()
      ? moment(report.updatedAt)
      : moment(report.releasedAt)
```
Or a more informative approach would be to display `createdAt` `updatedAt` together when not published:

> Authored: 16 November 2020 @ 10:27 Last Updated: 16 November 2020 @ 12:44  [Draft]

Reply:
I think before a report is published, it may not be important to distinguish creation date vs update date. @VassilIordanov i will go with this:
```
    const timeToShow = !report.isPublished()
      ? moment(report.updatedAt)
      : moment(report.releasedAt)
```
unless you have preference